.github/workflows/tests.yml

```diff
@@ -10,9 +10,9 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Install Dafny
-        uses: tchajed/setup-dafny-action@v1
+        uses: dafny-lang/setup-dafny-action@v1
         with:
-          dafny-version: "3.2.0"
+          dafny-version: "3.4.0"
```

Comment:
Can we make this a matrix to ensure support of more than just the latest Dafny? Given fixing verification between versions can be tough, it would be great for the standard library to not assume all users can easily move themselves.

Reply:
Changing the expect file is a breaking change, so we don't have compatibility to any previous version.